src/main/java/application/MenuController.java

```diff
@@ -65,10 +70,38 @@ public SessionBean performInstall(@RequestBody InstallRequestBean installRequest
     @ApiOperation(value = "Make the given menu selection and return the next set of options, or a form to play.")
     @RequestMapping(value = Constants.URL_MENU_SELECT, method = RequestMethod.POST)
     public SessionBean selectMenu(@RequestBody MenuSelectBean menuSelectBean) throws Exception {
+        log.info("Select Menu with bean: " + menuSelectBean);
         MenuSession menuSession = getMenuSession(menuSelectBean.getSessionId());
         boolean redrawing = menuSession.handleInput(menuSelectBean.getSelection());
         menuRepo.save(menuSession.serialize());
-        return getNextMenu(menuSession, redrawing);
+        SessionBean nextMenu = getNextMenu(menuSession, redrawing);
+        log.info("Returning menu: " + nextMenu);
+        return nextMenu;
+    }
+
+    /**
+     * Make a a series of menu selections (as above, but can have multiple)
+     *
+     * @param menuSelectRepeater Give the selections to be made on the current MenuSession
+     *                             (could be a module, form, or case selection)
+     * @return A MenuResponseBean or a NewFormSessionResponse
+     * @throws Exception
+     */
+    @ApiOperation(value = "Make the given menu selection and return the next set of options, or a form to play.")
+    @RequestMapping(value = Constants.URL_MENU_SELECT_REPEATER, method = RequestMethod.POST)
+    public SessionBean selectMenuRepeater(@RequestBody MenuSelectRepeater menuSelectRepeater) throws Exception {
```

Comment:
a lot of this logging seems like it should be `debug` not `info`, but don't have too strong of feelings